docs/readthedocs/source/doc/UserGuide/contributor.rst

```diff
@@ -0,0 +1,5 @@
+Contributor Guide
+=========================
+
+* `Developer Guide <./develop.html>`_ provides instructions to edit BigDL source code
```

Comment:
`Developer Guide` provides instructions on how to build from source and contribute to BigDL source code.